app/main.py

```diff
@@ -16,4 +16,13 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables = [lucky_number, pi, one_is_a_prime_number, name,
+             my_favourite_films, profile_info, marks, collection_of_coins]
+sorted_variables = {
```

Comment:
The `sorted_variables` should be a dictionary, not a set. Change the curly braces `{}` to square brackets `[]` to correctly define it as a dictionary.